automation/check-patch.e2e-k8s.sh

```diff
@@ -11,7 +11,7 @@ teardown() {
     make cluster-down
     cp $(find . -name "*junit*.xml") $ARTIFACTS
     # Don't fail if there is no logs
-    cp ${E2E_LOGS}/*.log ${ARTIFACTS} || true
+    cp ${E2E_LOGS}/handler/*.log ${ARTIFACTS} || true
```

Comment:
Not related to the PR

Reply:
Well without prow artifacts it's difficult to do forensics at prow.